synapse/lib/storm.py

```diff
@@ -5971,3 +5971,90 @@ async def execStormCmd(self, runt, genr):
                         path = runt.initPath(node)
                         path.vars.update(pathvars.get(buid))
                         yield (node, path)
+
+class StormPoolSetCmd(Cmd):
+    '''
+    Setup a Storm query offload mirror pool for the Cortex.
+    '''
+    name = 'storm.pool.set'
+    def getArgParser(self):
+        pars = Cmd.getArgParser(self)
+        pars.add_argument('--connection-timeout', type='int', default=2,
+            help='The maxiumem amount of time to wait for a connection from the pool to become available.')
+        pars.add_argument('--sync-timeout', type='int', default=2,
+            help='The maxiumem amount of time to wait for a connection from the pool to become available.')
+        pars.add_argument('url', type='str', required=True, help='The telepath URL for the AHA service pool. Use $lib.null to unset.')
```

Comment:
I don't think $lib.null to unset will work

Reply:
whups, vestigial before i implemented the del